docs/user-guides/observability-ui-plugins.md

```diff
@@ -0,0 +1,23 @@
+# Observability UI Plugins
+
+Using the Observability UI, you can install and manage plugins that extend the observability functionality of the OpenShift web console. Plugins are installed and managed by the Observability Operator.
+
+## Plugins
+
+- [dashboards](#dashboards): Add enhanced dashboards to the OpenShift web console. This plugin allows you to add other prometheus datasources present in the cluster, apart from the in-cluster one, to the default dashboards.
```

Comment:
> This plugin allows you to add other prometheus datasources present in the cluster, apart from the in-cluster one, to the default dashboards.

nit: Prometheus should be probably with capital P, but maybe it would be good to explicitly distinguish the groups - `moitoring.rhobs` vs `monitoring.coreos.com`